### PR TITLE
Hulu

### PR DESCRIPTION
**Pull request by trmcnvn**
- [x] Tested on Android
- [x] Tested on Physical device
- [x] Remote HTML moved to Kitsu and URL updated

### CODE REVIEW

**Comment by Mikunj** [comment]
Just curious on the case when the video is not available in a certain region, what would happen in the app?

**Comment by trmcnvn** [comment]
That depends on the API, if our API works correctly and doesn't return the results for non-supported regions then the video area isn't rendered, otherwise Hulu will just show it's not available splash screen

File: src/screens/Profiles/MediaPages/pages/Episodes/Unit/component.js [hunk 1]
```diff
@@ -206,11 +196,9 @@ class Unit extends PureComponent {
               style={styles.webContainer}
               // @TODO: replace with a Kitsu-based link
               source={{ uri: 'https://reminiscent-team.surge.sh' }}
-              onMessage={this.onMessage}
               renderLoading={this.renderLoading}
               renderError={this.renderError}
-              // This ensures `postMessage` has been patched by React-Native
-              injectedJavaScript="window.initializeHulu();"
+              injectedJavaScript={injectedJavaScript}
```
Comment (Mikunj) [inline src/screens/Profiles/MediaPages/pages/Episodes/Unit/component.js]: If there's a state update while a user is watching a video, does this affect the web view?

Comment (trmcnvn) [reply src/screens/Profiles/MediaPages/pages/Episodes/Unit/component.js]: Nope, not during my tests at least. (Delaying a state change until after video loads)

**Pull request merged**